e2e-tests/playwright/support/ui/components/channels/emoji_gif_picker.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, Locator} from '@playwright/test';
+
+export default class EmojiGifPicker {
+    readonly container: Locator;
+
+    readonly gifTab: Locator;
+    readonly gifInput: Locator;
+    readonly gifPlane: Locator;
+
+    constructor(container: Locator) {
+        this.container = container;
+
+        this.gifTab = container.getByText('GIFs');
+        this.gifInput = container.getByPlaceholder('Search GIPHY');
+        this.gifPlane = container.locator('.gif-picker__items')
```

Comment:
Instead of `gifPlane`, maybe `gifPickerItems`?

Reply:
Done ✅